src/Notify/Client/NotificationClient.cs

```diff
@@ -102,11 +103,30 @@ public TemplateList GetAllTemplates(String templateType = "")
             return templateList;
         }
 
-        public SmsNotificationResponse SendSms(String mobileNumber, String templateId, Dictionary<String, dynamic> personalisation = null, String clientReference = null)
+        public ReceivedTextListResponse GetReceivedTexts(String olderThanId = "")
+        {
+            String finalUrl = String.Format(
+                "{0}{1}",
+                GET_RECEIVED_TEXTS_URL,
+                String.IsNullOrWhiteSpace(olderThanId) ? "" : "?older_than=" + olderThanId
+            );
+
+            String response = this.GET(finalUrl);
```

Comment:
Should we check for a http response code i.e. 200 here?

Reply:
We don't do a check on the http response anywhere else, would mean a major refactor?